libs/auth/src/index.ts

```diff
@@ -9,3 +9,10 @@ export { DaffLoginRequest } from './models/login-request'
 export { DaffLoginResponse } from './models/login-response'
 
 export { DaffAuthModule } from './auth.module';
+
+export { DaffRegisterDriver } from './drivers/injection-tokens/register-driver.token';
+export { DaffRegisterServiceInterface } from './drivers/interfaces/register-service.interface';
+
+export { DaffLoginDriver } from './drivers/injection-tokens/login-driver.token';
+export { DaffLoginServiceInterface } from './drivers/interfaces/login-service.interface';
+
```

Comment:
need to export the tokens and interfaces for the transformer and the query manager